vb_suite/indexing.py

```diff
@@ -227,10 +226,10 @@
 series_loc_list_like = Benchmark("s.loc[[800000]]", setup)
 series_loc_array = Benchmark("s.loc[np.arange(10000)]", setup)
 
-series_iloc_scalar = Benchmark("s.loc[800000]", setup)
-series_iloc_slice = Benchmark("s.loc[:800000]", setup)
-series_iloc_list_like = Benchmark("s.loc[[800000]]", setup)
-series_iloc_array = Benchmark("s.loc[np.arange(10000)]", setup)
+series_iloc_scalar = Benchmark("s.iloc[800000]", setup)
```

Comment:
With this typo, ``Benchmark("series_iloc_scalar")``'s checksum would be same as ``Benchmark("series_loc_scalar")`` one, and which causes UNIQUE constraint error on database.